templates/partners/become-a-partner.html

```diff
@@ -0,0 +1,484 @@
+                  <option value="NV">Nevada</option>
+                  <option value="NY">New York</option>
+                  <option value="OH">Ohio</option>
+                  <option value="OK">Oklahoma</option>
+                  <option value="OR">Oregon</option>
+                  <option value="PA">Pennsylvania</option>
+                  <option value="PR">Puerto Rico</option>
+                  <option value="RI">Rhode Island</option>
+                  <option value="SC">South Carolina</option>
+                  <option value="SD">South Dakota</option>
+                  <option value="TN">Tennessee</option>
+                  <option value="TX">Texas</option>
+                  <option value="UT">Utah</option>
+                  <option value="VA">Virginia</option>
+                  <option value="VT">Vermont</option>
+                  <option value="WA">Washington</option>
+                  <option value="WI">Wisconsin</option>
+                  <option value="WV">West Virginia</option>
+                  <option value="WY">Wyoming</option>
+                </select>
+              </div>
+            </div>
+            <p class="p-heading--three">Partner programme</p>
+            <label for="interestSelect">Interested in</label>
+            <select name="interestSelect" id="interestSelect">
+              <option value="" disabled="disabled" selected="">Select...</option>
+              <option value="1">Cosmic Cuttlefish</option>
+              <option value="2">Bionic Beaver</option>
+              <option value="3">Xenial Xerus</option>
+            </select>
+            <textarea id="partbershipText" rows="5" placeholder="Please tell us more about the opportunity you see in partnering with Canonical..."></textarea>
+            <input type="checkbox" id="CanonicalUpdatesOptIn">
+            <label for="CanonicalUpdatesOptIn"><small>I agree to receive information about Canonical’s products and services.</small></label>
+            <p>
+              <small>In submitting this form, I confirm that I have read and agree to <a href="https://ubuntu.com/legal/data-privacy/newsletter">Canonical’s Privacy Notice</a> and <a href="https://ubuntu.com/legal/data-privacy">Privacy Policy</a>.</small>
+              </p>
+            <button class="p-button--positive" type="submit">Apply now</button>
+          </fieldset>
+        </form>
+      </div>
+    </div>
+  </section>
+  <section class="p-strip--image is-dark">
+    <div class="p-strip--suru-bottom">
+      <div class="row">
+        <div class="col-8">
+          <h1>Find a partner</h1>
+          <p class="u-sv3">Canonical’s network of Ubuntu partners spans the full range of technology activities. We offer partner programmes for software vendors, hardware OEMs, mobile carriers, channel partners, retailers and public cloud providers.</p>
+        </div>
+        <div class="col-4 u-align--center u-vertically-center">
```

Comment:
I think this image could do with being hidden on small screens

Reply:
@anthonydillon makes sens. I will hide it.